src/main/java/hu/wurfel/new_school_reference/subject_teacher/SubjectTeacherService.java

```diff
@@ -27,29 +27,29 @@ public SubjectTeacher toEntity(SubjectTeacherDto dto) {
 
 	//region Subject
 
-	public List<SubjectTeacher> findAllBySubject(Long id) {
-		return repo.findAllBySubject_IdAndDeletedIsFalse(id);
+	public List<SubjectTeacher> findAllBySubject(Long id, boolean deleted) {
+		return repo.findAllBySubject_IdAndDeleted(id, deleted);
 	}
 
-	public List<SubjectTeacher> findAllBySubjectName(String name) {
-		return repo.findAllBySubject_NameAndDeletedIsFalse(name);
+	public List<SubjectTeacher> findAllBySubjectName(String name, boolean deleted) {
```

Comment:
And again
`findAllBySubjectNameAndDeleted`
Vagy talán, ami célszerűbb: 
`findAllByDeletedAndSubjectName`
mert a deleted a SubjectTeacher fieldje, a name meg a Subjectté.